api/spec/controllers/mno_enterprise/pages_controller_spec.rb

```diff
@@ -27,6 +27,19 @@ module MnoEnterprise
       end
     end
 
+    describe 'GET #launch with parameters' do
+      let(:app_instance) { build(:app_instance) }
+      before { sign_in user }
+      subject { get :launch, id: app_instance.uid, specific_parameters: 'specific_parameters_value' }
+
+      it_behaves_like "a navigatable protected user action"
+
+      it 'redirect to the mno enterprise launch page with a web token' do
```

Comment:
could have been named `'redirects to the mno enterprise launch page with a web token and extra params'` :)